gourmet/gglobals.py

```diff
@@ -19,7 +19,7 @@
         # http://stackoverflow.com/questions/2608200/problems-with-umlauts-in-python-appdata-environvent-variable
         # We might drop this workaround with Python 3 (all strings are unicode)
         # and/or GTK+ 3 (use Glib.get_home_dir()).
-        APPDATA = windows.getenv('APPDATA').decode('utf-8')
+        APPDATA = windows.getenv('APPDATA')
```

Comment:
this shouldn't be part of the PR

Reply:
Otherwise I could not run the tests.

Reply:
That might be, but you don't need to commit all your changes.